### ISSUE TITLE
about section text renders outside the browser

### ISSUE THREAD

**Issue by Athanasiamo** [hivickylai/hugo-theme-introduction#31]
I love this theme, but am having an issue with the text from my about.md not wrapping properly when the browser size is above a certain size (about tablet size is when it starts behaving strangely).

![image](https://user-images.githubusercontent.com/14014329/37108038-3df36dc4-2236-11e8-93da-6e63410739bb.png)

wraps just fine when smaller
![image](https://user-images.githubusercontent.com/14014329/37108089-63aeb21c-2236-11e8-9b46-f59fe39c4f6f.png)

and from ipad it looks rather strange:
![image](https://user-images.githubusercontent.com/14014329/37108164-98448038-2236-11e8-8be7-18f3411de423.png)

**Comment by Athanasiamo** [comment]
I made some changes via the sass, but none that should (technically) make any difference, but I'm sure you're right. The sample text in the md wasn't long enough to pose a problem, so I added another sentence, and it wraps perfectly fine with the original theme. So there must be something I changed (ugh). 

Sorry about that, I should have increased the original text before I made a ticket.

Let's see if I can manage to debug this!
